File: src/ToDo/Models/Usuario.cs
using System.ComponentModel.DataAnnotations;

namespace ToDo.Models
{
    public class Usuario
    {
        [Required]
        [Key]
        public virtual int Id { get; set; }
        [Required]
        public virtual string Nome { get; set; }
        [Required]
        public virtual string Email { get; set; }

        public virtual ICollection<Tarefa> Tarefas { get; set; }
    }
}

File: src/todo-front/src/models/Usuario.ts
import { Tarefa } from "./Tarefa";

// src/models/Usuario.ts
export class Usuario {
  id?: number; // Opcional
  nome: string;
  email: string;
  senha: string;
  
  constructor(nome: string, email: string,senha: string, id?: number) {
      this.nome = nome;
      this.email = email;
      this.senha = senha;
      if (id !== undefined) {
          this.id = id; // Define apenas se fornecido
      }
  }
}